src/shared/lib/IMobXApplicationDataStore.ts

```diff
@@ -28,14 +28,18 @@ export class SimpleMobXApplicationDataStore<T> implements IMobXApplicationDataSt
     @observable protected dataHighlighter:(d:T)=>boolean;
 
     @observable public filterString:string;
-    @observable public sortMetric:SortMetric<T>;
-    @observable public sortAscending:boolean;
+    @observable public sortMetric:SortMetric<T>|undefined;
+    @observable public sortAscending:boolean|undefined;
 
     @computed get allData() {
         return this.data;
     }
     @computed get sortedData() {
-        return lazyMobXTableSort(this.allData, this.sortMetric, this.sortAscending);
+        // if not defined, use default values for sortMetric and sortAscending
+        const sortMetric = this.sortMetric || (() => 0);
+        const sortAscending = this.sortAscending !== undefined ? this.sortAscending : true;
```

Comment:
Actually, another option is to return `allData` when undefined. But then, we need to update several test cases for this class.